analyzer/runtime/extract.go

```diff
@@ -617,6 +642,57 @@ func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Ad
 				extractedEvents = append(extractedEvents, &eventData)
 				// .To is from another chain, so exclude?
```

Comment:
I did not follow this pattern with (un)delegate events, and I'm tempted to undo it for these "old" events to, i.e. I'm tempted to always extract `.To` as a related address. There's not really a downside to overindexing, other than bloating our DB a bit, as far as I know. And it might come handy to be able to look up `consensusaccounts` events related to a certain consensus account.

Reply:
ya ok

Reply:
After more discussion on Slack, we can't think of reasons why this would be problematic. So I'm adding `.To` (for `Withdraw` event) and `.From` (for `Deposit`) to the list of event-related addrs.